.make/targets-all.mk

```diff
@@ -0,0 +1,86 @@
+SHELL := /bin/bash
+
+DOCKER_NAME      := arrow-benchmarks-$(PACKAGE_NAME)
+IMAGE_NAME       := benchmarks-$(PACKAGE_NAME)
+BUILD_IMAGE_NAME := ghcr.io/arrow-air/tools/arrow-rust
+BUILD_IMAGE_TAG  := latest
+DOCKER_PORT      := 8000
+HOST_PORT        := 8080
+
+SOURCE_PATH      ?= $(PWD)
+
+# Style templates for console output.
+GREEN  := $(shell echo -e `tput setaf 2`)
+YELLOW := $(shell echo -e `tput setaf 3`)
+CYAN   := $(shell echo -e `tput setaf 6`)
+NC     := $(shell echo -e `tput setaf 9`)
+BOLD   := $(shell echo -e `tput bold`)
+SMUL   := $(shell echo -e `tput smul`)
+SGR0   := $(shell echo -e `tput sgr0`)
+
+# function with a generic template to run docker with the required values
+# Accepts $1 = command to run, $2 = additional flags (optional)
+docker_run = docker run \
+	--name=$(DOCKER_NAME)-$@ \
+	--rm \
+	--user `id -u`:`id -g` \
+	--workdir=/usr/src/app/$(PACKAGE_NAME) \
+	-e CARGO_INCREMENTAL=$(CARGO_INCREMENTAL) \
+	-e RUSTC_BOOTSTRAP=$(RUSTC_BOOTSTRAP) \
+	-v "$(SOURCE_PATH)/:/usr/src/app" \
+	-v "$(SOURCE_PATH)/.cargo/registry:/usr/local/cargo/registry" \
+	$(2) \
+	-t $(BUILD_IMAGE_NAME):$(BUILD_IMAGE_TAG) \
+	$(1)
+
+.SILENT: docker-pull
+
+help-all:
+	@echo ""
+	@echo "$(BOLD)$(CYAN)Available targets$(NC)$(SGR0)"
+	@echo ""
+	@echo "$(SMUL)$(BOLD)$(GREEN)TOML$(NC)$(SGR0)"
+	@echo "  $(BOLD)toml-test$(SGR0)   -- Run 'taplo format --check' to validate TOML file formats."
+	@echo "  $(BOLD)toml-tidy$(SGR0)   -- Run 'taplo format' to fix TOML file formats if needed."
+	@echo ""
+	@echo "$(SMUL)$(BOLD)$(GREEN)CSpell$(NC)$(SGR0)"
+	@echo "  $(BOLD)cspell-test$(SGR0) -- Run 'cspell --words-only --unique "**/**" -c .cspell.config.yaml'"
+	@echo "                 to validate files are not containing any spelling errors."
+	@echo "  $(BOLD)cspell-add-words$(SGR0) -- Run 'cspell --words-only --unique "**/**" -c .cspell.config.yaml | "
+	@echo "                      sort --ignore-case >> .cspell.project-words.txt'"
+	@echo "                      to add remaining words to the project's cspell ignore list"
+
+docker-pull:
+	@docker pull -q $(BUILD_IMAGE_NAME):$(BUILD_IMAGE_TAG)
+
+# TOML / taplo targets
+toml-test:
```

Comment:
docker pull here? And in the other toml check

Reply:
Gah, forgot the reason I was refactoring this bit and didn't check if `docker-pull` was now used everywhere it should.
Thx! It's fixed now